stream-helpers/mod.ts

```diff
@@ -0,0 +1,5 @@
+export * from "./batch.ts";
+export * from "./valve.ts";
+export * from "./map.ts";
+export * from "./filter.ts";
+export * from "./tracker.ts";
```

Comment:
We should either add jsdocs to tracker, or leave it out  or put an `@ignore` tag on its apis if we need it to test streams outside of the stream helpers package itself. As is, it will negatively effect our package score.

Reply:
Done